app-office/libalkimia/libalkimia-9999.ebuild

```diff
@@ -12,6 +12,7 @@ HOMEPAGE="http://kde-apps.org/content/show.php/libalkimia?content=137323"
 
 LICENSE="LGPL-2.1"
 KEYWORDS=""
+SLOT="0/6"
```

Comment:
this should be accompanied by blocking libalkimia:5, it seems you could just edit the existing (bogus) blocker on slot 4